microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -73,6 +73,23 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
 
 ![A possible issue being explained on an output element](microsoft-edge-devtools-extension-images/live-issue-reporting.png)
 
+#### Automated fixes and issue filtering
+
+When you hover over the element with the issue, you get a lightbulb icon indicating that there are quick fixes available.
+
+![An anchor element with a protocol specific href attribute highliighted as a problem with a lightbulb icon above it](microsoft-edge-devtools-extension-images/light-bulb.png)
```

Comment:
@JasonAndrewWriter @codepo8 @Reezaali - This .png shows "example.com", might be a requirement to change it to "contoso.com" and change demo code to that.  Next .png too.

Reply:
I don't have a button to Resolve conversation, and I don't want to Delete this comment which was substantive.
Pls Resolve the present conversation.